### PR TITLE
Backport and align files in release-4.6 wrt master

### PR DESCRIPTION
**Pull request by yussufsh**

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/263#" title="Author self-approved">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/release-4.6/OWNERS)~~ [yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: var.tfvars [hunk 1]
```diff
@@ -17,12 +17,6 @@ bootstrap                   = {memory      = "32",   processors  = "0.5",  "coun
 master                      = {memory      = "32",   processors  = "0.5",  "count"   = 3}
 worker                      = {memory      = "32",   processors  = "0.5",  "count"   = 2}
 
-### This is a sample config for trying out Cloud Pak for Data 
-#bastion                     = {memory      = "16",   processors  = "1",    "count"   = 1}
-#bootstrap                   = {memory      = "32",   processors  = "0.5",  "count"   = 1}
-#master                      = {memory      = "32",   processors  = "0.5",  "count"   = 3}
-#worker                      = {memory      = "64",   processors  = "1.5",  "count"   = 4}
-
```
Comment (bpradipt) [inline var.tfvars:25]: We should retain this.

File: docs/quickstart.md [hunk 2]
```diff
@@ -15,24 +15,14 @@
 
 ## Download the Automation Code
 
-Head to the [release](https://github.com/ocp-power-automation/ocp4-upi-powervs/releases) page and download the latest stable release.
-Extract the release bundle on your system.
+You'll need to use git to clone the deployment code when working off the master branch
 
-You can also use `curl` or `wget` to download the stable release code as shown below.
 ```
```
Comment (bpradipt) [inline docs/quickstart.md:20]: Shouldn't we have both the options (curl and wget) documented  ?

Comment (yussufsh) [reply docs/quickstart.md:20]: @bpradipt This is to align the docs as per master branch. IMO `git clone` is enough as we mentioned Git as a per-requisite.

File: var.tfvars (hunk 1 above)
Comment (yussufsh) [reply var.tfvars:25]: @bpradipt This is not there in master and other release branches as well. Made this change for the cherrypick bot to work without conflicts.

Comment (bpradipt) [reply var.tfvars:25]: ok

File: docs/quickstart.md (hunk 2 above)
Comment (bpradipt) [reply docs/quickstart.md:20]: That might be an oversight. Will approve this to go through.  In subsequent PRs will add the mention of downloading via curl or wget

**Review by bpradipt — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/263#pullrequestreview-633204924" title="LGTM">bpradipt</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/263#" title="Author self-approved">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/release-4.6/OWNERS)~~ [bpradipt,yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by cs-zhang — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/263#pullrequestreview-633204924" title="LGTM">bpradipt</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/263#pullrequestreview-633923286" title="LGTM">cs-zhang</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/263#" title="Author self-approved">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/release-4.6/OWNERS)~~ [bpradipt,yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**